pkg/trace/api/api.go

```diff
@@ -552,7 +534,15 @@ func (r *HTTPReceiver) handleGetConfig(w http.ResponseWriter, req *http.Request)
 		http.Error(w, err.Error(), http.StatusBadRequest)
 		return
 	}
-	cfg, err := r.coreClient.ClientGetConfigs(r.coreClientCtx, &configsRequest)
+	token, err := security.FetchAuthToken()
+	if err != nil {
+		killProcess("could obtain the auth token for the tracer remote config client: %v", err)
```

Comment:
Actually FetchAuthToken should be in the init. If the feature is enabled, a AuthToken should be available and could be stored in the HTTPReceiver.

Else the err handling will kill the process on one request, while the err could be caught at startup